docs/source/markdown/podman-create.1.md

```diff
@@ -364,6 +364,13 @@ The following example maps uids 0-2000 in the container to the uids 30000-31999
 
 Add additional groups to run as
 
+#### **--groups**=*false*
```

Comment:
Suggest rename to `--keep-groups` - that makes it much more clear what's going on.

Reply:
I hate options with embeded "-", why isn't --groups good enough, this is certainly what I would look for if I had this issue.

--groups=all or --groups=eng,staff would be my preference.

Reply:
I find `--groups` very confusing, because we already have `--group-add` which is quite similar but works on groups in the container. We need to distinguish between the two somehow.